sdk/samples/iot/generate_certificate.ps1

```diff
@@ -4,6 +4,11 @@
 Set-StrictMode -Version 3.0
 $errorActionPreference="stop"
 
+if($PSVersionTable.PSEdition -ne "Core")
+{
+ Write-Error "This script requires Powershell Core. Please install Powershell Core at aka.ms/pscore6."
```

Comment:
I don't think `Write-Error` is enough - there's potential for this to get lost with other stuff script puts out.  We also should have a `throw` to bring the whole thing to a halt.